pipelines/_ingredients/ApTemplateCalibrateImage.yaml

```diff
@@ -0,0 +1,80 @@
+description: The AP template building pipeline, with the new CalibrateImageTask.
+# Look in subdirectories of $AP_PIPE_DIR/pipelines to find customized pipelines
+# for each camera. Those pipelines import this general template-building pipeline.
+
+# WARNING: some camera-specific pipelines importing this pipeline presently
+# blow away all the configs that are set in this file.
+# To update a pipeline config prior to DM-35504, you MUST put it in either,
+# e.g., $AP_PIPE_DIR/config/$CAMERA/someTask.py, or in a camera-specific,
+# pipeline, e.g., $AP_PIPE_DIR/pipelines/$CAMERA/ApTemplate.yaml.
+
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcd.yaml
```

Comment:
```suggestion
  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcdCalibrateImage.yaml
```

Reply:
Thinking about it more: I'm going to remove this new ApTemplate pipeline. We need to confirm that we can run something like DRP with the new task first, instead of just trying to make templates with it (which I haven't done on this ticket).